overdrive.py

```diff
@@ -871,26 +871,28 @@ def __init__(self, _db, input_identifier_types=None,
             workset_size=10, metadata_replacement_policy=metadata_replacement_policy, **kwargs
         )
 
-    def process_batch(self, identifiers):
-        return [self.process_item(identifier) for identifier in identifiers]
-
-
     def process_item(self, identifier):
         info = self.api.metadata_lookup(identifier)
+        error = None
         if info.get('errorCode') == 'NotFound':
-            e = "ID not recognized by Overdrive"
-            return CoverageFailure(self, identifier, e, transient=False)
+            error = "ID not recognized by Overdrive: %s" % identifier.identifier
+        elif info.get('errorCode') == 'InvalidGuid':
+            error = "Invalid Overdrive ID: %s" % identifier.identifier
```

Comment:
This is just to turn an annoying Overdrive error from a transient failure to a permanent failure.